addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -0,0 +1,613 @@
+            json_payload.update({
+                "TransId": invoice.l10n_in_transporter_id.vat,
+                "TransName": invoice.l10n_in_transporter_id.name,
+            })
+        elif invoice.l10n_in_mode == "1":
+            json_payload.update({
+                "TransMode": invoice.l10n_in_mode,
+                "VehNo": invoice.l10n_in_vehicle_no,
+                "VehType": invoice.l10n_in_vehicle_type,
+            })
+        elif invoice.l10n_in_mode in ("2", "3", "4"):
+            doc_date = invoice.l10n_in_transportation_doc_date
+            json_payload.update({
+                "TransMode": invoice.l10n_in_mode,
+                "TransDocDt": doc_date and doc_date.strftime("%d/%m/%Y") or False,
+                "TransDocNo": invoice.l10n_in_transportation_doc_no,
+            })
+        return json_payload
+
+    def _l10n_in_edi_ewaybill_post_invoice_edi(self, invoices):
+        response = {}
+        res = {}
+        generate_json = self._l10n_in_edi_ewaybill_generate_json(invoices)
+        response = self._l10n_in_edi_ewaybill_generate(invoices.company_id, generate_json)
+        if response.get("error"):
+            error = response["error"]
+            error_codes = [e.get("code") for e in error]
+            if "238" in error_codes:
+                # Invalid token eror then create new token and send generate request again.
+                # This happen when authenticate called from another odoo instance with same credentials (like. Demo/Test)
+                authenticate_response = self._l10n_in_edi_ewaybill_authenticate(invoices.company_id)
+                if not authenticate_response.get("error"):
+                    error = []
+                    response = self._l10n_in_edi_ewaybill_generate(invoices.company_id, generate_json)
+                    if response.get("error"):
+                        error = response["error"]
+                        error_codes = [e.get("code") for e in error]
+            if "604" in error_codes:
+                # Get E-waybill by details in case of E-waybill is already generated
+                # this happens when timeout from the Government portal but E-waybill is generated
+                response = self._l10n_in_edi_ewaybill_get_by_consigner(
+                    invoices.company_id, generate_json.get("docType"), generate_json.get("docNo"))
+                if not response.get("error"):
+                    error = []
+                    odoobot = self.env.ref("base.partner_root")
+                    invoices.message_post(author_id=odoobot.id, body=markupsafe.Markup(_(
+                        "Somehow this Ewaybill is generate in government portal before." \
+                        "<br/>Normally, this should not happen too often" \
+                        "<br/>Just verify value of invoice by enter details into government website " \
+                        "<a href='https://ewaybillgst.gov.in/Others/EBPrintnew.aspx'>here<a>."
```

Comment:
try staying consistent (E-waybill) + check english + saying it should not happen too often is just confessing we don't have a clue and doesn't sound professional
```suggestion
                        "Somehow this E-waybill has been generated in the government portal before." \
                        "<br/>You can verify the details of the invoice by checking into the government website " \
                        "<a href='https://ewaybillgst.gov.in/Others/EBPrintnew.aspx'>here<a>."
```